pkg/server/api.go

```diff
@@ -0,0 +1,117 @@
+package server
```

Comment:
Although the code is quite self-explanatory, some comments would be "nice to have".